addOns/pscanrules/CHANGELOG.md

```diff
@@ -13,7 +13,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
     - User Controllable JavaScript Event (XSS)
     - Cookie without SameSite Attribute
     - X-Debug-Token Information Leak
-- The Cookie without SameSite Attribute scan rule now has alert references (Issue 7100).
+    - Retrieved from Cache
+- The following scan rules now have alert references (Issue 7100):
+    - Cookie without SameSite Attribute 
+    - Retrieved from Cache (invalid raw text was also trimmed from the Alert's references (Issue 8262))
```

Comment:
- Not sure about calling it invalid raw text, we didn't have any rules about what the references could have or not.
- Should be used singular, only one reference was changed.

Reply:
Okay, I had said "invalid" because it was making the new reference testing fail. I've tweaked it.